Tests/MarkdownTests/Parsing/BlockDirectiveParserTests.swift

```diff
@@ -1238,4 +1239,34 @@ class BlockDirectiveArgumentParserTests: XCTestCase {
         """#
         XCTAssertEqual(expected, document.debugDescription(options: .printSourceLocations))
     }
+    #endif
+}
+
+#if compiler(>=6.0) && canImport(Testing)
+import Testing
+
+struct _BlockDirectiveArgumentParserTests {
+    @Test(
+        "Directive MultiLine WithoutContent Parsing",
+        .bug("https://github.com/swiftlang/swift-markdown/issues" ,id: "#152", "Verify fix of #152")
```

Comment:
```suggestion
        .bug("https://github.com/swiftlang/swift-markdown/issues/152" ,id: "#152", "Verify fix of #152")
```

Reply:
Thought it was a base URL to replace .github/.radar etc.

Checked the full [prososal](https://github.com/swiftlang/swift-testing/blob/main/Documentation/Proposals/0001-refactor-bug-inits.md) and got it.